.github/workflows/release.yml

```
@@ -19,61 +19,25 @@ jobs:
+      - name: Initial checkout
         uses: actions/checkout@v3
-        with:
-          fetch-depth: 0 # required for getting commit history
 
-      - name: Set up build cache
-        uses: actions/cache@v3
+      - name: Common Checkout and Cache
+        uses: ./.github/actions/common-checkout-cache
         with:
-          path: |
+          branch: main
```

actionlint (1.7.4)

input "branch" is not defined in action "Common Checkout and Cache" defined at "./.github/actions/common-checkout-cache". available inputs are "cache-key-prefix", "cache-paths"

(action)